build/teamcity/cockroach/nightlies/roachtest_nightly_impl.sh

```diff
@@ -32,6 +32,8 @@ source $root/build/teamcity/util/roachtest_util.sh
 # Standard release branches are in the format `release-24.1` for the
 # 24.1 release, for example.
 release_branch_regex="^release-[0-9][0-9]\.[0-9]"
+# Test selection is enabled only on release branches.
+selective_tests="false"
```

Comment:
Fix for "build/teamcity/cockroach/nightlies/roachtest_nightly_impl.sh: line 76: selective_tests: unbound variable" when attempting to run SELECT_PROBABILITY on (this) PR branch.

Reply:
🤦 Ah yeah that broke one the last second change to early fail in the script itself. Should we pull this out as it's own fix, given other people are likely to run into this?

Reply:
> Should we pull this out as it's own fix so we can merge sooner, given other people are likely to run into this? 

Yeah, I was feeling lazy :) Will send a PR in a few.

Reply:
https://github.com/cockroachdb/cockroach/pull/137931